packages/block-library/src/query-pagination-next/index.php

```diff
@@ -44,7 +44,7 @@ function render_block_core_query_pagination_next( $attributes, $content, $block
 		$content = get_next_posts_link( $label, $max_page );
 		remove_filter( 'next_posts_link_attributes', $filter_link_attributes );
 	} elseif ( ! $max_page || $max_page > $page ) {
-		$custom_query           = new WP_Query( build_query_vars_from_query_block( $block, $page ) );
+		$custom_query           = new WP_Query( gutenberg_build_query_vars_from_query_block( $block, $page ) );
```

Comment:
I'll update this soon, because block files are copied as they are in the build process. The PR can be tested though as is.